src/main/java/n3/arrays/ArrayTask1.java

```diff
@@ -0,0 +1,19 @@
+package n3.arrays;
+
+public class ArrayTask1 {
+    public static void main(String[] args) {
+
+        //
+        String [] arr = {"tfjf", "txt", "wekjfwer", "frfr", "nnnnn", "r", "txt", "ewf", "ttt", "txt", "txt"};
+        String a = "txt";
+        int counter = 0;
+
+        for (String i:arr) {
+
+            if (i == a){
```

Comment:
You should read about how to compare 2 Strings for equality. It is definitely not operation '='.

Reply:
fixed, 
m-d .equals() is used to compare the values